### PR TITLE
69 tutorial raw readers

### PR DESCRIPTION
**Pull request by jalew188**
1. docs/tutorials will contain all the tutorial notebooks.
2. Add sphinx rst file for tutorial notebooks.
3. Add base settings and raw reader notebooks

### CODE REVIEW

File: tests/download_test_data.py [hunk 1]
```diff
@@ -13,10 +15,11 @@ def download_file(url, local_filename):
                 f.write(chunk)
 
 
+raw_dir = "../nbs_tests/test_data"
 url_template = (
```
Comment (GeorgWa) [inline tests/download_test_data.py:19]: I created a PR with the OOP test data downloader from alphaDIA.
I think this is something we could share.

File: docs/tutorials/spectrum_peak_df_connection.png
Comment (GeorgWa) [inline docs/tutorials/spectrum_peak_df_connection.png:1]: very nice

**Review by GeorgWa — APPROVED** [review]
LGTM

File: tests/download_test_data.py (hunk 1 above)
Comment (jalew188) [reply tests/download_test_data.py:19]: We are going to discuss this in alphaX meeting

**Pull request merged**